spec/system/admin/adding_to_an_order_spec.rb

```diff
@@ -185,4 +185,34 @@
     end
   end
 
+  describe "adding a product from another facility", :js do
+    let(:facility2) { create(:setup_facility) }
+    let(:product) { create(:setup_item, :with_facility_account) }
+    let!(:product2) { create(:setup_item, :with_facility_account, facility: facility2) }
+
+    before do
+      visit facility_order_path(facility, order)
+    end
+
```

Comment:
How about adding an accessibility scan here:
```
    it "is accessible" do
      expect(page).to be_axe_clean
    end
```
If it passes or there are simple fixes to make it pass, great.  If not, you can limit the scope checked by the spec (see [docs](https://docs.deque.com/devtools-html/4.0.0/en/rb-rspec)) and then we can improve it later on